CHANGELOG.md

```diff
@@ -26,6 +59,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   version conflicts caused by the inter-dependency between prost and tonic.
 - Python components exposes `pythonVersion`.
 - All derivation shells gets default shellCommands.
+- `callComponent <path>` which works like `callFile` but expect result to be a component
+  and creates one if it is not.
```

Comment:
This should be in unreleased, not version 7.0.0

Reply:
Oops